tests/test_hyp3_floods.py

```diff
@@ -84,20 +84,24 @@ def test_lambda_handler_missing_env_var():
         hyp3_floods.lambda_handler(None, None)
 
 
-def test_process_active_hazard_submit():
+@patch('hyp3_floods.get_aoi')
+def test_process_active_hazard_submit(mock_get_aoi: MagicMock):
     mock_hyp3 = NonCallableMock(hyp3_floods.HyP3SubscriptionsAPI)
     mock_hyp3.get_subscriptions_by_name.return_value = {'subscriptions': []}
     mock_hyp3.submit_subscription.return_value = {'subscription': {'subscription_id': ''}}
 
+    mock_get_aoi.return_value = 'POINT(48.0 38.0)'
```

Comment:
It might be worth converting all the points to polygons in these tests as that's what we'll typically expect now.

Reply:
That's a fair suggestion but since we don't actually do anything with the mock AOI strings (they could be any arbitrary string), I think I'll just leave it as-is.

Reply:
On second-thought, it should be a 1-minute fix so I will go ahead and do it.